pkg/mount/dag_steps.go

```diff
@@ -826,25 +902,22 @@ func (s *State) UKIBootInitDagStep(g *herd.Graph) error {
 					}
 					// If the directory has the same device as its parent, it's not a mount point.
 					if fileInfo.Sys().(*syscall.Stat_t).Dev == parentInfo.Sys().(*syscall.Stat_t).Dev {
-						internalUtils.Log.Debug().Msg(fmt.Sprintf("%s is a simple directory", path))
+						internalUtils.Log.Debug().Str("what", path).Msg("simple directory")
```

Comment:
is the "what" maybe a leftover?

Reply:
nope, thats what the log takes becuase it outputs json formatted ususally

so you can send it to somewhere and then use some search shit on it, structured logs and whatnot :D

looks like this on "normal" output (missing colors which are nice:

```
2024-03-14T20:48:13Z DBG simple directory what=system
2024-03-14T20:48:13Z DBG mount point what=tmp
2024-03-14T20:48:13Z DBG simple directory what=usr
2024-03-14T20:48:13Z DBG mount point what=var
2024-03-14T20:48:13Z DBG Mount moved from=/dev to=/sysroot/dev
2024-03-14T20:48:13Z DBG Mount moved from=/efi to=/sysroot/efi
2024-03-14T20:48:13Z DBG Mount moved from=/etc to=/sysroot/etc
2024-03-14T20:48:13Z DBG Mount moved from=/home to=/sysroot/home
2024-03-14T20:48:13Z DBG Mount moved from=/oem to=/sysroot/oem
2024-03-14T20:48:13Z DBG Mount moved from=/opt to=/sysroot/opt
2024-03-14T20:48:13Z DBG Mount moved from=/proc to=/sysroot/proc
2024-03-14T20:48:13Z DBG Mount moved from=/root to=/sysroot/root
2024-03-14T20:48:13Z DBG Mount moved from=/run to=/sysroot/run
2024-03-14T20:48:13Z DBG Mount moved from=/snap to=/sysroot/snap
2024-03-14T20:48:13Z DBG Mount moved from=/srv to=/sysroot/srv
2024-03-14T20:48:13Z DBG Mount moved from=/sys to=/sysroot/sys
2024-03-14T20:48:13Z DBG Mount moved from=/tmp to=/sysroot/tmp
2024-03-14T20:48:13Z DBG Mount moved from=/var to=/sysroot/var
```

Reply:
Gotcha!